salt/utils/templates.py

```diff
@@ -345,11 +346,17 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
                                        **env_args)
 
     jinja_env.filters['strftime'] = salt.utils.date_format
-    jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
     jinja_env.filters['yaml_dquote'] = salt.utils.yamlencoding.yaml_dquote
     jinja_env.filters['yaml_squote'] = salt.utils.yamlencoding.yaml_squote
     jinja_env.filters['yaml_encode'] = salt.utils.yamlencoding.yaml_encode
 
+    for function_name in context['salt'].keys():
```

Comment:
Which is why I am asking if there is a better way to force the execution modules to load. That is the only reason this loop exists.

At the moment, the primary improvement is that this implementation works for `utils` functions.